### PR TITLE
FactorsAI doc

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

Created FactorsAI doc

### CODE REVIEW

File: docs/destinations/streaming-destinations/factors-ai.mdx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+A sample `track` call is shown below:
+
+```javascript
+rudderanalytics.track("Product Reviewed", {
+  review_id: "12345",
+  product_id: "123",
+  rating: 3.0,
+  review_body: "Average product, expected much more."
+})
+```
```
Comment (ItsSudip) [inline docs/destinations/streaming-destinations/factors-ai.mdx:61]: We need to add one FAQ section where we have to provide the information to get the API key. I have updated the notion card with details.

**Review by ItsSudip — APPROVED** [review]

**Review by ameypv-rudder — APPROVED** [review]

File: docs/destinations/streaming-destinations/factors.mdx [hunk 2]
```diff
@@ -0,0 +1,70 @@
+---
+title: "FactorsAI"
+description: Step-by-step guide on sending your event data from RudderStack to FactorsAI.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/factors.mdx:3]:
```suggestion
description: Step-by-step guide on sending your event data from RudderStack to Factors.
```

File: docs/destinations/streaming-destinations/factors.mdx [hunk 3]
```diff
@@ -0,0 +1,70 @@
+---
+title: "FactorsAI"
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/factors.mdx:2]:
```suggestion
title: "Factors"
```

File: docs/destinations/streaming-destinations/factors.mdx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+description: Step-by-step guide on sending your event data from RudderStack to FactorsAI.
+aliases: ["Factors AI destination"]
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/factors.mdx:4]:
```suggestion
aliases: ["Factors destination"]
```

File: docs/destinations/streaming-destinations/factors.mdx [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+---
+
+[FactorsAI](https://www.factors.ai/) is a marketing analytics and attribution platform. It collects and processes all the data across your customer journeys to help you make smart business decisions.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/factors.mdx:7]:
```suggestion
[Factors](https://www.factors.ai/) is a marketing analytics and attribution platform. It collects and processes all the data across your customer journeys to help you make smart business decisions.
```

File: docs/destinations/streaming-destinations/factors.mdx [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+<div class="infoBlock">
+Find the open source code for this destination in the <a href="https://github.com/rudderlabs/rudder-transformer/tree/master/v0/destinations/factorsai">GitHub repository</a>.
+</div>
+
+## Getting started
+
+RudderStack supports sending event data to FactorsAI via the following <a href="https://rudderstack.com/docs/rudderstack-cloud/rudderstack-connection-modes/">connection modes</a>:
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/factors.mdx:15]:
```suggestion
RudderStack supports sending event data to Factors via the following <a href="https://rudderstack.com/docs/rudderstack-cloud/rudderstack-connection-modes/">connection modes</a>:
```

File: docs/destinations/streaming-destinations/factors.mdx [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+| **Connection Mode** | **Web**       | **Mobile**    | **Server**    |
+| :------------------ | :------------ | :------------ | :------------ |
+| **Device mode**     | -             | -             | -             |
+| **Cloud mode**      | **Supported** | **Supported** | **Supported** |
+
+Once you have confirmed that the source platform supports sending events to FactorsAI, follow these steps:
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/factors.mdx:22]:
```suggestion
Once you have confirmed that the source platform supports sending events to Factors, follow these steps:
```

File: docs/destinations/streaming-destinations/factors.mdx [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+
+1. From your [RudderStack dashboard](https://app.rudderstack.com/), add the source. Then, from the list of destinations, select **FactorsAI**.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/factors.mdx:24]:
```suggestion
1. From your [RudderStack dashboard](https://app.rudderstack.com/), add the source. Then, from the list of destinations, select **Factors AI**.
```

File: docs/destinations/streaming-destinations/factors.mdx [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+2. Assign a name to your destination and click **Continue**.
+
+## Connection settings
+
+To successfully configure FactorsAI as a destination, you will need to configure the following settings:
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/factors.mdx:29]:
```suggestion
To successfully configure Factors as a destination, you will need to configure the following settings:
```

File: docs/destinations/streaming-destinations/factors.mdx [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+
+- **API Key**: Enter the FactorsAI API Key. For more information on obtaining your FactorsAI API key, refer to the <Link to="#faq">FAQ</Link> section below.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/factors.mdx:31]:
```suggestion
- **API Key**: Enter the Factors API Key. For more information on obtaining your FactorsAI API key, refer to the <Link to="#faq">FAQ</Link> section below.
```

File: docs/destinations/streaming-destinations/factors.mdx [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+
+## Identify
+
+You can use the <Link to="/event-spec/standard-events/identify/">`identify`</Link> call to create or update a user in FactorsAI. RudderStack uses the <code class="inline-code">userId</code> field to perform these actions.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/factors.mdx:35]:
```suggestion
You can use the <Link to="/event-spec/standard-events/identify/">`identify`</Link> call to create or update a user in Factors. RudderStack uses the <code class="inline-code">userId</code> field to perform these actions. 
```

File: docs/destinations/streaming-destinations/factors.mdx [hunk 12]
```diff
@@ -0,0 +1,70 @@
...
+
+```javascript
+rudderanalytics.identify('1hKOmRA4e', {
+  email: 'alex@example.com'
+});
+```
+
+## Track
+
+You can use the <Link to="/event-spec/standard-events/track">`track`</Link> call to capture user events and their associated properties, and send this information to FactorsAI.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/factors.mdx:47]:
```suggestion
You can use the <Link to="/event-spec/standard-events/track">`track`</Link> call to capture user events and their associated properties, and send this information to Factors.
```

File: docs/destinations/streaming-destinations/factors.mdx [hunk 13]
```diff
@@ -0,0 +1,70 @@
...
+  review_id: "12345",
+  product_id: "123",
+  rating: 4.0,
+  review_body: "Good product."
+})
+```
+
+## FAQ
+
+### Where can I get the FactorsAI API key?
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/factors.mdx:62]:
```suggestion
### Where can I get the Factors API key?
```

File: docs/destinations/streaming-destinations/factors.mdx [hunk 14]
```diff
@@ -0,0 +1,70 @@
...
+
+To get your Factors API key:
+
+1. Log in to your [FactorsAI dashboard](https://app.factors.ai/).
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/factors.mdx:66]:
```suggestion
1. Log in to your [Factors dashboard](https://app.factors.ai/).
```

File: docs/destinations/streaming-destinations/factors.mdx [hunk 15]
```diff
@@ -0,0 +1,70 @@
...
+2. Click the menu button in the left corner and go to **Settings** > **Integrations**.
+3. From the integrations list, select **RudderStack**  to generate the API key, as shown:
+
+<img src="../../assets/event-stream-destinations/factorsai-api-key.png" alt="FactorsAI API key" />
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/factors.mdx:70]:
```suggestion
<img src="../../assets/event-stream-destinations/factorsai-api-key.png" alt="Factors API key" />
```

**Pull request merged**